### PR TITLE
build and scan e2e-test-runner image

### PR DESCRIPTION
**Pull request by xinredhat**
Added e2e-test-runner image required to run the build and scan CI

### CODE REVIEW

File: .github/workflows/build-push-images.yaml [hunk 1]
```diff
@@ -56,6 +56,10 @@ jobs:
             vulnerability:
               - '.github/workflows/build-push-images.yaml'
               - 'ci/images/vulnerability-scan/**'
+            e2e-test-runner:
```
Comment (Roming22) [inline .github/workflows/build-push-images.yaml:59]: Please try to keep list alphabetically ordered. It makes browsing through easier.

File: .github/workflows/individual-image-scanner-quay.yaml [hunk 2]
```diff
@@ -31,6 +31,7 @@ jobs:
       static-checks-output: ${{ steps.static-checks-scan.outputs.VULNERABILITIES_EXIST }}
       update-pipeline-service-output: ${{ steps.update-pipeline-service-scan.outputs.VULNERABILITIES_EXIST }}
       vulnerability-scan-output: ${{ steps.vulnerability-scan.outputs.VULNERABILITIES_EXIST }}
+      e2e-test-runner-output: ${{ steps.e2e-test-runner-scan.outputs.VULNERABILITIES_EXIST }}
```
Comment (Roming22) [inline .github/workflows/individual-image-scanner-quay.yaml:34]: Same comment about ordered lists.

File: .github/workflows/individual-image-scanner-quay.yaml [hunk 3]
```diff
@@ -64,6 +65,9 @@ jobs:
               - 'shared/**'
             vulnerability:
               - 'ci/images/vulnerability-scan/**'
+            e2e-test-runner:
```
Comment (Roming22) [inline .github/workflows/individual-image-scanner-quay.yaml:68]: Same comment about ordered lists.

File: .github/workflows/periodic-scanner-quay.yaml [hunk 4]
```diff
@@ -32,6 +32,7 @@ jobs:
             "static-checks"
             "update-pipeline-service"
             "vulnerability-scan"
+            "e2e-test-runner"
```
Comment (Roming22) [inline .github/workflows/periodic-scanner-quay.yaml:35]: Same comment about ordered lists.

File: .github/workflows/build-push-images.yaml (hunk 1 above)
Comment (xinredhat) [reply .github/workflows/build-push-images.yaml]: fixed

File: .github/workflows/individual-image-scanner-quay.yaml (hunk 2 above)
Comment (xinredhat) [reply .github/workflows/individual-image-scanner-quay.yaml]: fixed

File: .github/workflows/individual-image-scanner-quay.yaml (hunk 3 above)
Comment (xinredhat) [reply .github/workflows/individual-image-scanner-quay.yaml]: fixed

File: .github/workflows/periodic-scanner-quay.yaml (hunk 4 above)
Comment (xinredhat) [reply .github/workflows/periodic-scanner-quay.yaml]: fixed

**Review by Roming22 — APPROVED** [review]

**Pull request merged**